README.md

```diff
@@ -120,7 +120,3 @@ The services are DIM RPC services. Every RPC is called with a string and expects
 * Example:
   * DIM input `0x3\n`
   * DIM output ` `
-
-#### Examples
```

Comment:
Why remove the examples? While these are certainly not useful for the end-user me or a future maintainer should know where to look.

Reply:
Concern that it encourages users to interact directly with ALF, which will not be permitted in production. I would suggest perhaps a separate technical documentation for this kind of detail.

Reply:
Fair enough.